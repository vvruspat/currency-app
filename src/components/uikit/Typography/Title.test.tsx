import { render, screen } from "@testing-library/react";
import { Title } from ".";

test("renders title", () => {
  render(<Title>Test title</Title>, {});
  const titleElement = screen.getByText(/Test title/i);
  expect(titleElement).toBeInTheDocument();
  expect(titleElement).toMatchSnapshot();
});
