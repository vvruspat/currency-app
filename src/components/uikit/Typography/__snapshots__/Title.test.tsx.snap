// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders title 1`] = `
<div
  class="title"
>
  Test title
</div>
`;
